stock_picking_operation_quick_change/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+====================================
+Stock Picking Operation Quick Change
+====================================
+
+This module extends standard WMS to allow change destination location for all
+picking operations.
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to products and create one of type "Stockable".
+#. Update quantities on hand to have stock of it.
+#. Go to inventory dashboard and click on "Delivery Orders" card to do a new
+   transfer.
+#. Create a picking and select the product to do the transfer.
+#. Reserve the picking.
+#. Click on button *Change Location*, select in wizard the old location and
+   new location.
```

Comment:
Please reproduce this steps yourself to test the module

Reply:
Updated. Now this button appears :)

Reply:
:wink: